api/features/tests/test_tasks.py

```diff
@@ -51,13 +51,12 @@ def test_trigger_feature_state_change_webhooks(MockThread):
     event_type = environment_webhook_call_args[1]["args"][2]
     assert data["new_state"]["feature_state_value"] == new_value
     assert data["previous_state"]["feature_state_value"] == initial_value
-    assert event_type == WebhookEventType.FLAG_UPDATED
+    assert event_type == WebhookEventType.FLAG_UPDATED.value
```

Comment:
Same as above, can we add a check here for environment_id/organisation_id

Reply:
Done, thanks!